src/components/MovieList/MovieItem/MovieItem.tsx

```diff
@@ -8,47 +8,45 @@ import {
 } from "./MovieItem.styled";
 import ContextMenu from "../../ContextMenu/ContextMenu";
 import { Mode, Movie } from "../../../models/movie";
-import { useAppContext } from "../../../hooks/useAppContext";
+import { useAppDispatch } from "../../../hooks/hooks";
```

Comment:
No action needed: if you're tired of endless `../` check out this https://create-react-app.dev/docs/importing-a-component/#absolute-imports